cli/glnexus_cli.cc

```diff
@@ -698,17 +705,26 @@ int main_unify_sites(int argc, char *argv[]) {
 
     static struct option long_options[] = {
         {"help", no_argument, 0, 'h'},
+        {"bed", required_argument, 0, 'b'},
```

Comment:
Please add this to the usage message, mentioning it should be the same ranges provided to discover_alleles.